HelloWorld_ST_Sensors/main.cpp

```diff
@@ -117,71 +112,74 @@ int main()
     if (is_recv)
     {
       is_recv = false;
       printf("btrecv: %s\r\n", btserial_data);
     }
 
+    debugPrintf("ST B-L475E-IOT01A - I2C Slave\r\n");
+
     // Internal Sensor data
     internal_sensor::update_sensor_data();
     const internal_sensor::data_s &data = internal_sensor::get_sensor_data();
-    debugPrintf("HTS221: [temp] %.2f C, [hum] %.2f\r\n",
+    debugPrintf("HTS221: Temperature(C):          %.2f     | Humidity: %.2f\r\n",
                 data.hts221_temperature, data.hts221_humidity);
-    debugPrintf("LPS22HB: [temp] %.2f C, [press] %.2f mbar\r\n",
+    debugPrintf("LPS22HB: Temperature(C):         %.2f     | Pressure(mbar): %.2f\r\n",
                 data.lps22hb_temperature, data.lps22hb_pressure);
-    debugPrintf("LIS3MDL [mag/mgauss]:    %6ld, %6ld, %6ld\r\n",
+    debugPrintf("LIS3MDL: Magnetic Field(mgauss): X: %6ld | Y: %6ld | Z: %6ld\r\n",
                 data.magnetometer_axes[0], data.magnetometer_axes[1],
                 data.magnetometer_axes[2]);
-    debugPrintf("LSM6DSL [acc/mg]:        %6ld, %6ld, %6ld\r\n",
+    debugPrintf("LSM6DSL: Acceleration(mg):       X: %6ld | Y: %6ld | Z: %6ld\r\n",
                 data.acceleration_axes[0], data.acceleration_axes[1],
                 data.acceleration_axes[2]);
-    debugPrintf("LSM6DSL [gyro/mdps]:     %6ld, %6ld, %6ld\r\n",
+    debugPrintf("LSM6DSL: Gyroscope(mdps):        X: %6ld | Y: %6ld | Z: %6ld\r\n",
                 data.gyroscope_axes[0], data.gyroscope_axes[1],
                 data.gyroscope_axes[2]);
-    debugPrintf("VL53L0X [mm]:            %6ld\r\n", data.distance);
+    debugPrintf("VL53L0X: Time of Flight(mm): %7ld\r\n", data.distance);
 
     // CO Sensor
-    debugPrintf(
-        "SPEC CO SENSOR: Gas Concentration (ppm) %lu Temperature (C) %d "
-        "(Relative Humidity) %d\r\n",
-        co.get_gas_concentration(), co.get_temperature(),
-        co.get_relative_humidity());
+    uint32_t spec_co_gas_conc = co.get_gas_concentration();
+    int16_t spec_co_temp = co.get_temperature();
+    uint16_t spec_co_rel_humidity = co.get_relative_humidity();
 
-    // SGP30 Sensor
-    debugPrintf("SGP30: (co2) %d (voc) %d\r\n", sgp30.get_co2(),
-                sgp30.get_voc());
+    debugPrintf("IOT_CO_1000_CO:                  Gas Conc.(PPM): %4lu | Temperature(C): %d | "
+                "Relative Humidity: %u\r\n",
+                spec_co_gas_conc, spec_co_temp,
+                spec_co_rel_humidity);
 
     // GPS
     if (is_gps_recv == true)
     {
-      debugPrintf("Lat: %lf | Long: %lf\r\n", gps_coordinates_data.latitude, gps_coordinates_data.longitude);
+      debugPrintf("AP6H GPS:                        Latitude(deg): %f | Longitude(deg): %f\r\n", gps_coordinates_data.latitude, gps_coordinates_data.longitude);
       is_gps_recv = false;
     }
-    gps_coordinates_data.latitude = 0.0;
-    gps_coordinates_data.longitude = 0.0;
+
+    // Heartbeat Sensor
+    const heartbeat::bioData hb_data = heartbeat::get_hb_data();
```

Comment:
Is the heartbeat giving you data since you changed the API?
It will now be `hb_obj.get_hb_data();`